README.md

```diff
@@ -82,4 +82,10 @@ Here are some SQL queries for most common use cases
 
 ![SQL Query1](resources/screenshots/query1.png)
 
-![SQL Query2](resources/screenshots/query2.png)
\ No newline at end of file
+![SQL Query2](resources/screenshots/query2.png)
+
+### DBT dag diagram - Visual representation of the data transformation workflow
+![dbt dag diagram](resources/screenshots/dbt-dag.png)
+
+### The chart represents the cumulative number of stars received by each repository over time. It allows us to see the growth in popularity of each repository over time.
+![Stars per month growth for each repo](resources/screenshots/stars_by_month.png)
```

Comment:
Same here – would suggest moving images to the top of the README.

Maybe put this under ## Sample Chart: Cumulative Stars per Repo